rspace/src/main/scala/coop/rchain/rspace/history/RadixTree.scala

```diff
@@ -201,7 +215,12 @@ object RadixTree {
           decodeItem(pos0Next, nodeNext) // Try to decode next item.
         }
 
-      decodeItem(0, emptyNode)
+      try {
+        decodeItem(0, emptyNode)
+      } catch {
+        case _: Exception =>
+          throw new Exception("Error during deserialization: invalid data format")
```

Comment:
Original exception must be supplied to wrapped exception. Please see my example in previous comment.